edx2bigquery/s3_backend.py

```diff
@@ -109,4 +113,67 @@ def get_tracking_log_objects(bucket_name, start_date):
             if not object_file:
                 continue
 
-            download_object_and_save(object_file.get('Key', None))
+            object_key = object_file.get('Key', '')
+            local_file_name = object_key.split('/')[-1]
+            local_path_name = '{}/{}'.format(
+                logs_dir,
+                local_file_name,
+            )
+            download_object_and_save(object_key, local_path_name)
+
+
+def get_sql_data_objects(bucket_name, start_date):
+    """
+    Gets the MYSQL data generated by edx-analytics-exporter from Amazon S3.
+
+    Args:
+        bucket_name: Name of the bucket to the get the MySQL data.
+        start_date: String date to find the MySQL data.
+    Raises:
+        Exception: If not bucket name was provided in the configuration file.
+    """
+    if not bucket_name:
+        raise Exception('AWS_BUCKET_NAME must be specified in the configuration file.')
+
+    # .zip Since edx-analytics-exporter uploads a .zip file containing all the generated data.
+    object_key_name = '{}/{}{}{}'.format(
+        getattr(edx2bigquery_config, 'SQL_DATA_BUCKET_PATH', ''),
+        getattr(edx2bigquery_config, 'SQL_FILE_NAME_PREFIX', ''),
+        start_date,
+        '.zip'
+    )
+    sql_data_local_dir = '{}/{}{}'.format(
+        getattr(edx2bigquery_config, 'SQL_LOCAL_FOLDER', ''),
+        getattr(edx2bigquery_config, 'SQL_FILE_NAME_PREFIX', ''),
+        start_date,
+    )
+    my_path = os.path.dirname(os.path.realpath(__file__))
+    parent_path = os.path.abspath(os.path.join(my_path, os.pardir))
+    path_to_extract = '{}/{}'.format(
+        parent_path,
+        getattr(edx2bigquery_config, 'SQL_SOURCE_DATA_LOCAL_FOLDER', ''),
+    )
+
+    if not os.path.exists(getattr(edx2bigquery_config, 'SQL_LOCAL_FOLDER', '')):
```

Comment:
constant

Reply:
This is a setting name, so, it's not neccesary.